fun/starboard.go.tmpl

```diff
@@ -31,7 +32,9 @@
 	{{ end }}
 {{ end }}
 
-{{ if and $count (or .ReactionMessage.Content .ReactionMessage.Attachments) (eq .Reaction.Emoji.Name $starEmoji) }}
+{{ if lt $count $starLimit }}
+	{{ deleteMessage $starboard $starboardMessage 0 }}
+{{ else if and $count (or .ReactionMessage.Content .ReactionMessage.Attachments) (eq .Reaction.Emoji.Name $starEmoji) (le (currentTime.Sub .Message.Timestamp.Parse) (toDuration $maxAge))}}
```

Comment:
```suggestion
{{ else if and $count (or .ReactionMessage.Content .ReactionMessage.Attachments) (eq .Reaction.Emoji.Name $starEmoji) ((.Message.Timestamp.Parse.Add (toDuration $maxAge)).Before currentTime)}}
```
; )

Reply:
:tomstare:
I guess I shouldn't be surprised